Header header
std_msgs/String path
std_msgs/Float64[] bounds